VideoLocker/src/main/java/org/edx/mobile/view/adapters/DiscussionCommentsAdapter.java

```diff
@@ -79,31 +89,30 @@ public void setProgressVisible(boolean visible) {
         if (viewType == RowType.PROGRESS) {
             return new RecyclerView.ViewHolder(LayoutInflater.
                     from(parent.getContext()).
-                    inflate(R.layout.list_view_footer_progress, parent, false)) {};
+                    inflate(R.layout.list_view_footer_progress, parent, false)) {
+            };
         }
 
-        return new ViewHolder(LayoutInflater.
+        @LayoutRes
+        int layout = R.layout.row_discussion_comments_comment;
```

Comment:
This assignment should be in the `else` clause I suppose. It can also be declared as `final` in that case.